src/app.rs

```diff
@@ -64,4 +76,70 @@ impl App {
 
         Ok(())
     }
+
+    /// # Errors
```

Comment:
It's better to use a `#[allow(clippy::...)] // TODO` exception than this. That way we can keep track of the todos